metafacture-xml/src/main/java/org/metafacture/xml/GenericXmlHandler.java

```diff
@@ -34,7 +34,11 @@
  * @author Markus Michael Geipel
  *
  */
-@Description("A generic xml reader")
+@Description("A generic xml reader. Separates XML data in distrinct records with the defined record tag name (default: `recordtagname=\"record\"`)" +
+        "If no matching record tag is found, the output will be empty." +
+        "The handler breaks down XML elements with simple string values and optional attributes" +
+        "into entities with a value subfield (name configurable) and additional subfields for each attribute." +
```

Comment:
```suggestion
@Description("A generic xml reader. Separates XML data in distrinct records with the defined record tag name (default: `recordtagname=\"record\"`) " +
        "If no matching record tag is found, the output will be empty. " +
        "The handler breaks down XML elements with simple string values and optional attributes " +
        "into entities with a value subfield (name configurable) and additional subfields for each attribute. " +
```

Add whitespaces as suggested by @blackwinter